assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -45,14 +45,27 @@ const Space = styled.div`
   flex: 0 0 10px;
 `;
 
-function PaddingControls({ selectedElements, pushUpdateForObject }) {
+function PaddingControls({
+  selectedElements,
+  pushUpdateForObject,
+  pushUpdate,
+}) {
   const padding = useCommonObjectValue(
     selectedElements,
     'padding',
     DEFAULT_PADDING
   );
+  const isPaddingLockOpen = getCommonValue(selectedElements, 'paddingLockOpen');
```

Comment:
Could we rename the stored value here? `paddingLockOpen` is a confusing name (it kind of reads like `padlock`, which has nothing to do with this). Maybe `paddingRatioLocked`, `hasFixedPaddingRatio` or something like that. Can't really come up with a good suggestion, though. @dvoytenko?

Reply:
Oh, and use the same name in the component as in the store - it's confusing if they're two different names.

Reply:
Used `lockPadding` and `isPaddingLock`. Let me know if you have any good suggestions for naming... 👍